platform/engineering/frontend/platform-assignments.md

```diff
@@ -0,0 +1,40 @@
+# Overview 
+
+`vets-website` has several parts that require ongoing maintenance and improvement. These platform parts are assigned a primary and a secondary engineer. 
+
+This document describes the responsibilities of each of these roles and is intended for VSP frontend engineers and their leadership group. 
+
+## Primary 
+
+- Accountable for the ongoing maintenance of their designated platform part 
+  - Expected to do around a day of unplanned maintenance per sprint- this does not include support time but should be documented in a ticket and draft PR
```

Comment:
I think we should list out examples of this so people understand what you mean.

- Fix identified bugs
- Improvements based on customer feedback
- Refactoring efforts
- Updates or integration work based on dependencies
- Improve clarity or scope of documentation based on feedback